docs/api/session.md

```diff
@@ -698,6 +698,43 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
 })
 ```
 
+#### `ses.setDisplayMediaRequestHandler(handler)`
+
+* `handler` Function | null
+  * `request` Object
+    * `frame` [WebFrameMain](web-frame-main.md) - frame that is requesting access to media.
+    * `securityOrigin` String - origin of the page making the request.
+    * `videoRequested` Boolean - true if the web content requested a video stream.
+    * `audioRequested` Boolean - true if the web content requested an audio stream.
+    * `userGesture` Boolean - whether a user gesture was active when this request was triggered.
+  * `callback` Function
+    * `streams` Object
+      * `video` Object | WebFrameMain (optional)
+        * `id` String - the id of the stream being granted.
+        * `name` String - the name of the stream being granted.
+      * `audio` Object | WebFrameMain (optional)
+        * `id` String - the id of the stream being granted.
+        * `name` String - the name of the stream being granted.
+
+This handler will be called when web content requests access to display media
```

Comment:
We should document specifically what happens if you pass a `webFrameMain` to the callback